src/_xoWithTestHelpers.js

```diff
@@ -59,4 +60,21 @@ export default class XoWithTestHelpers extends Xo {
     );
     this.backupNgIds.length = 0;
   }
+
+  async createTempVm(params) {
```

Comment:
Can we improve creating/deleting temporary resources to avoid code duplication?

Reply:
:+1:  Yes, I think we can do it.